po/es.po

```diff
@@ -19,441 +19,441 @@ msgstr ""
 
 #: ui/clapper.ui:6
 msgid "Open Files..."
-msgstr ""
+msgstr "Abrir archivos"
 
 #: ui/clapper.ui:10
 msgid "Open URI..."
-msgstr ""
+msgstr "Abrir URI"
```

Comment:
Missed three dots or was this on purpose?

Reply:
Was for a purpose.